### PR TITLE
Submit Arrow-Java for inclusion

### PR DESCRIPTION
**Pull request by liyafan82**
Arrow-Java is the Java implementation of Apache Arrow. 
This submission is for fuzzing the Arrow Java IPC reader.

### CODE REVIEW

**Comment by emkornfield** [comment]
Any reason not to add this to the existing ARrow project?

**Comment by liyafan82** [comment]
> Any reason not to add this to the existing ARrow project?

I am not sure if this is the best way to support Arrow-Java. I supported it in a separate folder because:

1. Each project requires a separate project.yaml file, which requires a single element in the "language" field.
2. It seems C++ and Java requires different base docker images.

**Comment by pitrou** [comment]
I find it reasonable to make it a separate project. This will also make life easier when browsing through detected issues. Arrow C++ maintainers won't act on Java issues, and vice-versa.

**Comment by jonathanmetzman** [comment]
Let me find out if we approve of adding this project

**Comment by jonathanmetzman** [comment]
We approve

File: projects/arrow-java/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Copyright 2021 Google LLC
```
Comment (asraa) [inline projects/arrow-java/Dockerfile:1]: nit: 2022 here and elsewhere

**Review by asraa — APPROVED** [review]

File: projects/arrow-java/Dockerfile (hunk 1 above)
Comment (liyafan82) [reply projects/arrow-java/Dockerfile]: Fixed. Thanks for your careful review. @asraa @jonathanmetzman

**Pull request merged**